bundle/internal/annotation/descriptor.go

```diff
@@ -9,6 +9,7 @@ type Descriptor struct {
 	MarkdownExamples    string `json:"markdown_examples,omitempty"`
 	DeprecationMessage  string `json:"deprecation_message,omitempty"`
 	Preview             string `json:"x-databricks-preview,omitempty"`
+	OutputOnly          *bool  `json:"x-databricks-field-behaviors_output_only,omitempty"`
```

Comment:
Question - do we serialize this type ever? or are this annotations just in case?

Why not keep them small, e.g. "output_only"?